### DISCUSSION TITLE
Installing CoApp on Ubuntu 24.04 not working

### DISCUSSION THREAD

**Discussion by technout** [aclap-dev/video-downloadhelper#434] [Help and Q&A] [answered]
### Your question

When installing the CoApp on x86_64 Ubuntu 24.04 LTS:
Did the install instructions and installed with this script:
curl -sSLf https://github.com/aclap-dev/vdhcoapp/releases/latest/download/install.sh | bash
After download error occurred!

Did manual download of het .tar.bz2 file and did the extract command (same error as in the script, see below):

user@pc:~$ tar -xf Downloads/vdhcoapp-linux-x86_64.tar.bz2 -C ~/.local/share/
tar (child): lbzip2: Cannot exec: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now

I did also cd into Downloads directory, but same error.
Please help :)

### Addon & CoApp version and details

Platform Linux x86-64
Browser Mozilla Firefox 125.0.2

Video DownloadHelper
Version 8.2.2.8 (stable)
Browser locale: en-US
Production build
Built on 2024-03-18
Build options: browser=firefox, linuxlic=false, noyt=false, target=mozilla

CoApp version: https://github.com/aclap-dev/vdhcoapp/releases/latest (v2.0.19)

**Comment by jcfrog** [comment]
Hi,
could you try to create the directory: 
`mkdir -p ~/.local/share`
The dev team will fix the script

**Comment by Season4episode10** [comment]
Am about to install Linux Lite (Ubuntu based and my go to Linux since it's initial release) on a laptop and spend as much spare time as I can afford to setup everything up, with Win10 going out of support next year, the constant downgrade in performance when a new major release is forced on my system and Win11' from all I've read being a total basterd I'm doing the hard work now as other video downloading software I pay for now works in Linux ... what we need, and I'm not the first to say this to this group is a group dedicated to supporting each other to make this something others can be helped through.

**Comment by technout** [comment]
> Hi, could you try to create the directory: `mkdir -p ~/.local/share` The dev team will fix the script

Thanks for the quick reply!
Unfortunately I still got the same error output:

Extracting tarball…
/tmp/vdhcoapp-JuPJQq: bzip2 compressed data, block size = 900k
tar (child): lbzip2: Cannot exec: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now

Edit: i did install Ubuntu 24.04 LTS on a new SSD and it was a fresh install (no upgrade). Also i choose for the default selection of tools and programs within the installation process.

**Comment by paulrouget** [comment, marked as answer]
Can you do that first?

```
sudo apt install bzip2
```

**Reply by technout** [reply]
Thanks! That solved the problem.
